server/routes/serviceProviderReferrals/serviceProviderReferralsController.test.ts

```diff
@@ -2155,6 +2155,61 @@ describe('GET /service-provider/referrals/:id/supplier-assessment/post-assessmen
   })
 })
 
+describe('GET /service-provider/referrals/:id/supplier-assessment/post-assessment-feedback', () => {
+  it('renders a page showing the supplier assessment feedback', async () => {
+    const deliusServiceUser = deliusServiceUserFactory.build()
+    const referral = sentReferralFactory.assigned().build()
+    const appointment = appointmentFactory.build({
+      appointmentTime: '2021-02-01T13:00:00Z',
+      sessionFeedback: {
+        attendance: {
+          attended: 'yes',
+          additionalAttendanceInformation: 'He was punctual',
+        },
+        behaviour: {
+          behaviourDescription: 'Acceptable',
+        },
+        submitted: false,
+      },
+    })
+    const supplierAssessment = supplierAssessmentFactory.build({
+      appointments: [appointment],
+      currentAppointmentId: appointment.id,
+    })
+    communityApiService.getServiceUserByCRN.mockResolvedValue(deliusServiceUser)
+    interventionsService.getSentReferral.mockResolvedValue(referral)
+    interventionsService.getSupplierAssessment.mockResolvedValue(supplierAssessment)
+
+    await request(app)
+      .get(`/service-provider/referrals/${referral.id}/supplier-assessment/post-assessment-feedback`)
+      .expect(200)
+      .expect(res => {
+        expect(res.text).toContain('View feedback')
+        expect(res.text).toContain('Did Alex attend the initial assessment appointment?')
+        expect(res.text).toContain('Yes, they were on time')
+        expect(res.text).toContain('Describe Alex&#39;s behaviour in this session')
+        expect(res.text).toContain('Acceptable')
+      })
+  })
+  it('renders an error if there is no current appointment for the supplier assessment', async () => {
```

Comment:
This is probably the worst comment I've ever left on a PR, and I wouldn't if I hadn't seen it a few times in this PR and in the codebase, but it's quite nice to leave a blank line between tests (particularly between `it`s or between `it`s and test setup).

...sorry. 

Don't worry about changing it in this PR, but just for future reference. I can also learn to live with it if you prefer it this way!

Reply:
😂

Reply:
Sure, I'll aim to do that in future!